src/System.Net.NetworkInformation/tests/FunctionalTests/StatisticsParsingTests.cs

```diff
@@ -224,23 +224,23 @@ public void IpInterfaceStatisticsParsingLast()
             FileUtil.NormalizeLineEndings("NetworkFiles/dev", fileName);
             IPInterfaceStatisticsTable table = StringParsingHelpers.ParseInterfaceStatisticsTableFromFile(fileName, "lo");
 
-            Assert.Equal(uint.MaxValue, table.BytesReceived);
```

Comment:
The old test is verifying when transferred bytes > `UInt32.MaxValue` (4294967295), `IPInterfaceStatisticsTable` cuts values to `uint.MaxValue`. We may also want to verify that the change follows this behavior (cut value to `(ulong)long.MaxValue`).

Reply:
Add this test back (but with long.MaxValue)